### PR TITLE
add some more temporary files to gitignore

### PR DESCRIPTION
**Pull request by mikedanese**
pesky temp files. any emacs users out there?

### CODE REVIEW

**Comment by ixdy** [comment]
e2e failed due to issue being fixed by #9744, sorry.

**Comment by mikedanese** [comment]
@k8s-bot ok to test

**Comment by mikedanese** [comment]
@nikhiljindal since you were the last to change `hooks/` do you mind reviewing

File: .gitignore [hunk 1]
```diff
@@ -58,3 +60,7 @@ www/master/shared/config/development.json
 
 # Karma output
 www/test_out
+
+# precommit temporary directories:
+_tmp/
+doc_tmp/
```
Comment (nikhiljindal) [inline .gitignore]: I will suggest adding a comment here regarding who creates this dir.

**Comment by nikhiljindal** [comment]
LGTM

**Pull request merged**